internal/signalfx-agent/bundle/Dockerfile

```diff
@@ -352,7 +352,7 @@ ENV useful_bins=" \
   /bin/ss \
   /bin/tar \
   /bin/umount \
-  /usr/bin/curl \
+  /usr/bin/wget \
```

Comment:
If this is regarding a low libcurl cve addressing that at the source would be my suggestion since we don't resolve it w/ this change.